aws_securityhub/plugin.spec.yaml

```diff
@@ -968,10 +968,10 @@ actions:
         type: object
         required: false
     output:
-      Findings:
+      findings:
```

Comment:
@mgorny-morecode We don't need to change the schema for these unless there's an issue with it. Otherwise, it will make the plugin backward-incompatible in workflows and requires a major version bump and users to re-enter their credentials.

The naming is fine here. We'll need to revert all these, and if done in other plugins.